lineapy/cli/cli.py

```diff
@@ -94,11 +94,10 @@
 @click.option(
     "--logging-level",
     type=click.Choice(
-        list(logging._nameToLevel.keys())
-        + sorted([str(x) for x in set(logging._levelToName.keys())]),
+        list(logging._nameToLevel.keys()),
```

Comment:
double checking that removing the sorted bit does not lead to issues/changes in behavior

Reply:
sounds good!